lib/assessment.js

```diff
@@ -8,15 +8,19 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  if (previousValue < currentValue) {
+    return previousValue;
+  } else {
+    return currentValue;
+  }
```

Comment:
Yep, this is right.

Here's how it could have been written more tersely.
```javascript
var min = function(a, b) {
  return (a < b)? a : b;
};
```